tests/accounts/base.py

```diff
@@ -19,6 +19,7 @@ class AccountTestCase(SitewitTestCase):
     password = 'password'
     user_token = 'user_token'
     url = 'http://www.test.site.com'
+    user_package = 'Free'
```

Comment:
Is there some agreement for package names? e.g. `free`  vs `Free` (personally I would prefer lower-cased version)

Reply:
Does it make sense in the context of the test data? Why it matters which case is used there?

Reply:
Answering the question directly: no, there is no agreement, need to clarify that.

Reply:
I'm not asking in the context of tests, sorry for confusing.  For tests it's totally fine to have anything (but even there I would prefer data which is similar (or same) to the real one.

Reply:
Unfortunately, this is the part that we need to investigate in https://github.com/yola/sitewitservice/issues/288